server/datastore/psql/maintenance_requests.go

```diff
@@ -0,0 +1,35 @@
+package psql
+
+import (
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+//AllMaintenanceRequests creates and returns a slice of datatype MaintenanceRequest with possible errors
+func (db *DB) AllMaintenanceRequests() ([]datastore.MaintenanceRequest, error) {
+	var maintenanceRequestTable []datastore.MaintenanceRequest
+	rows, err := db.connection.Query("SELECT * FROM maintenance_requests") //Number limits # of requests, can be anything
+	if err != nil {
+		return nil, err
+	}
+
+	defer rows.Close()
+	for rows.Next() {
+		// Create a new MaintenanceRequest
+		var maintenance_request datastore.MaintenanceRequest
+		// Scan the row values into MaintenanceRequest
+		err = rows.Scan(&maintenance_request.ID, &maintenance_request.Request, &maintenance_request.UserSubmitted, &maintenance_request.DateSubmitted)
+		//get any errors before the first iteration, checks if DB table is spelt correctly
+		if err != nil {
+			return maintenanceRequestTable, err
```

Comment:
thought I commented this before but golang standard practice is to return nil for the first return element if an err is present. So could you do this for everywhere there's an error returned?